multi contract/test/sample-test.js

```diff
@@ -1,5 +1,5 @@
 const { use } = require("chai");
-const { Contract, getAccountByName, junokitChai } = require("junokit");
+const { Contract, getAccountByName, trestleChai } = require("junokit");
```

Comment:
We'll be using junokitChai here as it has been renamed now.

Reply:
Yes already renamed in the fixes commit which is after entry point fix commit.